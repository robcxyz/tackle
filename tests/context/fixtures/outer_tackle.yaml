
outer:
  type: tackle inner_tackle.yaml


inner:
  type: print {{ outer }}
