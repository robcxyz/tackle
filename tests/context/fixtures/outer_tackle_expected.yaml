outer:

  stuff: things

  more_things: lots of things

  foo:
    - bar
    - baz

  foo_items:
    - bar
    - baz

  foo_index:
    - 0
    - 1
